web/sass-files/sass/partials/_modal.scss

```diff
@@ -213,8 +210,15 @@
 				right: -13px;
 				top: -13px;
 				@include opacity(0);
+                -webkit-transition: opacity 0.6s;
+                -moz-transition: opacity 0.6s;
+                -o-transition: opacity 0.6s;
+                transition: opacity 0.6s;
 				cursor: pointer;
 				z-index: 9999;
+                &.modal-close--show {
+				    @include opacity(1);
+                }
```

Comment:
Ugh... it's all indented using tabs. I overwrote my vimrc for the mm directory to only use spaces.  I guess we don't have a style guide for css though.